test/index.js

```diff
@@ -8,7 +8,7 @@ var m = require('../');
 require('co-mocha');
 
 // default test database
-var uri = process.env.YIELDB_TEST_URI;
+var uri = process.env.YIELDB_TEST_URI || 'mongodb://localhost/yieldb_test';
```

Comment:
You are right. Though I really wish one can do just these to run the test. Env variables are handled differently on different OS.

```
npm install
npm test
```